internal/pkg/version/check_remote.go

```diff
@@ -62,7 +62,9 @@ func (c *checker) CheckIfLatest(currentVersion string) error {
 	if latest.GreaterThan(current) {
 		c.logger.Warn(`*******************************************************`)
 		c.logger.Warnf(`WARNING: A new version "%s" is available.`, latestVersion)
+		c.logger.Warnf(`You are currently using the version "%s".`, current.String())
```

Comment:
Ten člen je tam nadbytečný když říkáš jakou verzi. (Leda že bys chtěl nějak extra zdůrazňovat že "opravdu tuto verzi", ale to imho není tento případ.)

Reply:
dik, fixnem